package.json

```diff
@@ -15,6 +15,16 @@
     "categories": [
         "Other"
     ],
+    "capabilities": {
```

Comment:
❤️❤️☕️ Nice, this is step forward for this repo, also if it is ok can we please make sure about this specific aspect in this codebase https://github.com/microsoft/vscode/wiki/Virtual-Workspaces#review-that-the-code-is-ready-for-virtual-resources 

Thank you so much.